test/unit/sql/insert.test.js

```diff
@@ -124,6 +124,32 @@ describe(Support.getTestDialectTeaser('SQL'), () => {
     });
   });
 
+  describe('time', () => {
+    it('formats time datatype correctly when inserting', () => {
```

Comment:
2 issues here:
1. it doesn't make sense to have `describe` for a single test (/ `it`).
2. The change doesn't belong to `insert`, so this shouldn't be a unit test there.  Instead it should be a unit test on data-types (file: [here](https://github.com/sequelize/sequelize/blob/main/test/unit/sql/data-types.test.js)) and (optionally) in integration tests for the places that implement it.

Reply:
I have removed the test from `insert`  and added test in `data-types.test.json ` .  Hope this is okay.